forge/routes/api/project.js

```diff
@@ -786,6 +786,15 @@ module.exports = async function (app) {
             settings.env = Object.assign({}, settings.settings.env, settings.env)
             delete settings.settings.env
         }
+
+        const ha = await request.project.getSetting('ha')
+        if (ha && ha.replicas > 1) {
+            settings.ha = ha
+            // not sure about this, is it something the launcher should do based on settings.ha being present?
+            // also doesn't disable the link in the FlowForge app
+            settings.settings.disableEditor = true
+        }
```

Comment:
```suggestion
        if (app.config.features.enabled('ha')) {
            const ha = await request.project.getHASettings()
            if (ha && ha.replicas > 1) {
                settings.ha = ha
            }
        }
```
We should push disabling the editor further down the stack. In the future we may be able to enable the editor in HA mode, but will depend on the NR version (I have the start of an idea for NR 4.x) - so we should push the decision further down.